src/arch/svm/utils.ts

```diff
@@ -507,6 +507,29 @@ export const getEmergencyDeleteRootBundleRootBundleId = (body: Buffer): number =
   return result.rootBundleId.toNumber();
 };
 
+/**
+ * Converts a common `RelayData` type to an SvmSpokeClient.RelayData` type. This is useful for when we need
+ * to interface directly with the SvmSpoke.
+ * @param relayData The common RelayData TS type.
+ * @returns RelayData which conforms to the typing of the SvmSpoke.
+ */
+export function toSvmRelayData(relayData: RelayData): SvmSpokeClient.RelayData {
```

Comment:
Moved this from the relayer https://github.com/across-protocol/relayer/blob/2a57f93cba62d64a82130ab2506aaabf0f29592b/src/utils/TypeUtils.ts#L18